knowledge-base/actions/irongut/editrelease/action-security.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Edit Release'
+# irongut/EditRelease
+github-token:
+  action-input:
+    input: token
+    is-default: true
+  permissions:
+    contents: write
+    issues-reason: to edit release details
```

Comment:
Thanks a lot @irongut for the PR! I believe the reason the tests are failing is that the reason should be `contents-reason` and not `issues-reason` since it is for the `contents` permission. Once you make that change, I think it should pass.